uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/drivers/roboteq_driver/src/CanopenInterface.cpp

```diff
@@ -97,8 +97,8 @@ CanopenInterface::CanopenInterface(canid_t roboteq_can_id, const std::string& if
   can_receive_filter[1].can_id = SDO_RESPONSE_COB_ID_OFFSET + roboteq::CanopenInterface::roboteq_can_id_;
   can_receive_filter[1].can_mask = (CAN_EFF_FLAG | CAN_RTR_FLAG | CAN_EFF_MASK);
 
-  //   struct timeval receive_timeout = {.tv_usec=500000}; //0.5 seconds
-  // setsockopt(socket_handle_, SOL_SOCKET, SO_RCVTIMEO, &receive_timeout, sizeof(receive_timeout));
+  struct timeval receive_timeout = {.tv_usec = 500000};  // 0.5 seconds
```

Comment:
unrelated to this pr:

oh @azumnanji. I understand why u were having compiler issues for this last week. Its because this syntax (i think its called designated initializers) is only valid in C99(and newer) or C++20. I mustve been using a compiler with C++20 enabled. We could just enable C++ 20 features?